pkg/core/system.go

```diff
@@ -282,6 +284,19 @@ func deleteNamespaces(kc *kubectlClient, namespaces []string) error {
 	return nil
 }
 
+func deleteSingleResource(kc *kubectlClient, resourceType string, name string) error {
```

Comment:
should also accept a namespace rather than assuming the default

Reply:
we don't know the namespace since we don't know what was the "default" namespace during install. Best we can do is hope that it is the same when run the uninstall.

Reply:
good point